lib/blocktree/blocktree.go

```diff
@@ -418,3 +418,10 @@ func (bt *BlockTree) GetBlockRuntime(hash common.Hash) (runtime.Instance, error)
 	}
 	return ins, nil
 }
+
+// ClearRuntimes deletes all runtime instances from BlockTree runtime map
+func (bt *BlockTree) ClearRuntimes() {
+	bt.runtimes.mutex.Lock()
+	bt.runtimes.mapping = make(map[Hash]runtime.Instance)
+	bt.runtimes.mutex.Unlock()
```

Comment:
This should be a method on `bt.runtimes` ideally since there are already a few methods on it, so that it would be easier to move it to a subpackage later.